core/src/UserSettingsEditor.html

```diff
@@ -78,6 +78,12 @@
 
     storedUserSettingData[userSettingGroup[0]][key] = option;
   }
+
+  function getPlaceholderValue(optionsEntry) {
+    const value = GenericHelpers.isObject(optionsEntry) ? Object.values(optionsEntry)[0] : optionsEntry;
+    
+     return $getTranslation(value);
+  }
```

Comment:
This is just a prettier auto formatting. It would be nice to add. Code looks good to me, works well 👍   
```suggestion

  function getPlaceholderValue(optionsEntry) {
    const value = GenericHelpers.isObject(optionsEntry)
      ? Object.values(optionsEntry)[0]
      : optionsEntry;

    return $getTranslation(value);
  }
```